### PR TITLE
Bug 2031793: Add validation for provided resource list in the migplan

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

File: pkg/controller/migplan/validation.go [hunk 1]
```diff
@@ -207,6 +208,12 @@ func (r ReconcileMigPlan) validate(ctx context.Context, plan *migapi.MigPlan) er
 		return liberr.Wrap(err)
 	}
 
+	// Hooks
+	err = r.validateInludedResources(ctx, plan)
```
Comment (sseago) [inline pkg/controller/migplan/validation.go:212]: Typo: s/Inluded/Included/

File: pkg/controller/migplan/validation.go [hunk 2]
```diff
@@ -221,6 +228,35 @@ func (r ReconcileMigPlan) validate(ctx context.Context, plan *migapi.MigPlan) er
 	return nil
 }
 
+func (r ReconcileMigPlan) validateInludedResources(ctx context.Context, plan *migapi.MigPlan) error {
```
Comment (sseago) [inline pkg/controller/migplan/validation.go:231]: Typo: s/Inluded/Included/

File: pkg/controller/migplan/validation.go [hunk 3]
```diff
@@ -207,6 +208,12 @@ func (r ReconcileMigPlan) validate(ctx context.Context, plan *migapi.MigPlan) er
 		return liberr.Wrap(err)
 	}
 
+	// Hooks
```
Comment (sseago) [inline pkg/controller/migplan/validation.go:211]: // Included resources

**Review by sseago — CHANGES_REQUESTED** [review]
A couple of minor changes (typo, copy-and-paste) but otherwise looks good.

File: pkg/controller/migplan/validation.go (hunk 1 above)
Comment (pranavgaikwad) [reply pkg/controller/migplan/validation.go:212]: I gotta take an eye exam :P

**Review by sseago — APPROVED** [review]

**Pull request merged**